e2e-tests/demand-backup-sharded/compare/find-2nd2.json

```diff
@@ -0,0 +1,4 @@
+switched to db myApp2
+{ "_id" : , "x" : 100500 }
+{ "_id" : , "x" : 100501 }
+bye
```

Comment:
Why we need two the same files?

Reply:
Not completely the same, it's one for each database `myApp`, `myApp1` and `myApp2`.

Reply:
I see your point, but that would require changing how compare function works and changing many compare files - let's not do it in this PR.

Reply:
ok, np.